internal/corazawaf/waf.go

```diff
@@ -256,35 +257,14 @@ func NewWAF() *WAF {
 		logger: &log.Logger{},
 		Level:  loggers.LogLevelInfo,
 	}
-	logWriter, err := loggers.GetLogWriter("serial")
-	if err != nil {
-		logger.Error("error creating serial log writer: %s", err.Error())
-	}
 	waf := &WAF{
 		// Initializing pool for transactions
 		txPool:                   sync.NewPool(func() interface{} { return new(Transaction) }),
-		ArgumentSeparator:        "&",
-		AuditLogWriter:           logWriter,
-		AuditEngine:              types.AuditEngineOff,
-		AuditLogParts:            types.AuditLogParts("ABCFHZ"),
-		RequestBodyAccess:        false,
-		RequestBodyInMemoryLimit: 131072,
-		RequestBodyLimit:         134217728, // 10mb
-		RequestBodyLimitAction:   types.BodyLimitActionReject,
-		ResponseBodyMimeTypes:    []string{"text/html", "text/plain"},
-		ResponseBodyLimit:        524288,
-		ResponseBodyLimitAction:  types.BodyLimitActionReject,
-		ResponseBodyAccess:       false,
-		RuleEngine:               types.RuleEngineOn,
```

Comment:
I agree about the point of reducing implicit defaults, I'm just wondering if it could be worth keeping here explicit the "base value". I'm thinking about boolean values, enums, maybe empty slices? Being programmatically redundant, maybe even just as a comment (but it could become easily outdated).

With this change running a base config becomes very needed, we are implicitly also changing defaults (e.g. Body limits now I think will default to ProcessPartial)

Reply:
I think at least `false` makes sense.

Reply:
The defaults should be loaded by https://github.com/corazawaf/coraza/pull/613